### ISSUE TITLE
Feature loadmore not work

### ISSUE THREAD

**Issue by MinhCuongIT** [alesdi/firestore-explorer#4]
Other features like reading and writing work as a charm. However with the button "Load 10 more", I cannot click and there is no option for me to load more than 10 items in a collection. Please check it. I am using Visual Studio Code for Windows 10.
Thanks a lot!

**Comment by Serzhmen** [comment]
Yep, beautiful extension, but this feature is not working, while it is needed badly...

**Comment by kailash-shewtahul** [comment]
You can enable this by modifying package.json.

line 94
`
  {
  "command": "firestore-explorer.showMoreItems",
  "title": "Show More Items",
  "enablement": "true"
  }
`
and line 127
`
  {
  "command": "firestore-explorer.showMoreItems",
  "when": "true"
  }
`
